### ISSUE TITLE
[BUG] Arrow keys wont work

### ISSUE THREAD

**Issue by Bizarrus** [microsoft/inshellisense#51]
**Describe the bug**
Arrow keys wont work

**To Reproduce**
1. Go to '[...](https://github.com/microsoft/inshellisense/tree/main)'
2. Install 'npm install -g @microsoft/inshellisense'
3. execute 'inshellisense bind'
4. select **bash**
5. Enter some commands
6. use arrow up or arrow down
7. Nothing happens

**Expected behavior**
```
 if (key.upArrow) {
    setSelectionIdx(Math.max(0, selectionIdx - 1));
}
else if (key.downArrow) {
    setSelectionIdx(Math.min(availableShells.length - 1, selectionIdx + 1));
}
```

Self-explanatory.I think `setSelectionIdx` will not executed, i dont know..

**Environment**
 - OS: Debian 12, `Linux my.fruit 5.15.90.1-microsoft-standard-WSL2 #1 SMP Fri Jan 27 02:56:13 UTC 2023 x86_64 GNU/Linux` 
 - Output of `inshellisense --version`: 0.0.1-rc.4 
 - Nodejs Version: node -v (v18.13.0), npm -v (9.2.0)

**Comment by cpendery** [comment]
Duplicate of #9

**Comment by cpendery** [comment]
Closing as this was fixed in [0.0.1-rc.5](https://github.com/microsoft/inshellisense/releases/tag/0.0.1-rc.5)
